src/main/java/edu/ucla/library/iiif/auth/services/DatabaseService.java

```diff
@@ -20,6 +20,16 @@
      */
     String ADDRESS = DatabaseService.class.getName();
 
+    /**
+     * The failure code if the service is unable to query the underlying database.
+     */
+    int INTERNAL_ERROR = 1;
```

Comment:
`INTERNAL_ERROR` and `NOT_FOUND_ERROR` looks like constants; shouldn't they be declared `static final`?

Reply:
Since they're declared on an interface, that's how they are by default.

If this _were_ a class, though, our automated code style checks would cause a build to ❌